english/empty-room/enter-room.md

```diff
@@ -0,0 +1,2 @@
+You find yourself in an empty room. Pitch black. Devoid of all light. You walk forward, making no sound.  Hearing nothing. Utter silence.  Darkness. Silence.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.